frontend/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.tsx

```diff
@@ -303,6 +303,24 @@ export function LinkWithTargetBlank(props: LinkProps): ReactElement {
   )
 }
 
+function makeMarkdownHeading(tag: string, markdown: string): string {
+  switch (tag.toLowerCase()) {
+    // willhuang1997: TODO: could be refactored to Enums
```

Comment:
I believe this would require a change in the protobuf. Since we released this, we probably won't change. That being said. This TODO is fine if we want to do a breaking change.

Reply:
Will 2 items to the tech debt db for follow up about finding a better place for this `Header` component and refactoring this code to Enums. Am I free to merge?